src/elements/common/sub-header/ViewModeChangeButton.js

```diff
@@ -22,18 +22,15 @@ type Props = {
 
 const ViewModeChangeButton = ({ className = '', onViewModeChange, intl, viewMode, ...rest }: Props) => {
     const isGridView = viewMode === VIEW_MODE_GRID;
-
+    const viewMessage = isGridView ? intl.formatMessage(messages.listView) : intl.formatMessage(messages.gridView);
```

Comment:
Can we use `<FormattedMessage />` instead, to avoid the `injectIntl` HOC?

Reply:
The HOC is already included and the Formatted Message component wont work with the ARIA labels.